spec/features/work_packages/progress_modal_spec.rb

```diff
@@ -528,6 +528,33 @@ def visit_progress_query_displaying_work_package
         # => % complete is derived
         progress_popover.expect_values(work: "20h", remaining_work: "4h", percent_complete: "80%")
       end
+
+      # scenario from https://community.openproject.org/wp/57370
+      specify "Case 33-2: when remaining work and % complete are cleared, " \
+              "changing or clearing work does not modify % complete at all" do
+        puts "#{Time.current.iso8601(3)} #{__FILE__}:#{__LINE__} "
```

Comment:
Is this temporary or supposed to stay in the spec?

Reply:
oups, it was me debugging an issue.
Thanks